### PR TITLE
PREAPPS-7985: Added dark mode support

### PR DESCRIPTION
**Pull request by lavegupta**

### CODE REVIEW

File: helpers.less [hunk 1]
```diff
@@ -234,4 +234,8 @@
 	}
 }
 
-
+.error() {
```
Comment (silentsakky) [inline helpers.less:237]: possible to do this using plain css?

Comment (lavegupta) [reply helpers.less:237]: No, Currently it is not possible with plain CSS.

**Review by silentsakky — APPROVED** [review]

**Review by k-kato — APPROVED** [review]

File: variables.less [hunk 2]
```diff
@@ -213,22 +218,22 @@
...
-@font-size-h3: calc(var(--font-size-base) * 1.7); // ~24px
-@font-size-h4: calc(var(--font-size-base) * 1.25); // ~18px
-@font-size-h5: var(--font-size-base);
-@font-size-h6: calc(var(--font-size-base) * 0.85); // ~12px
+@font-size-h1: var(--font-size-h1); // ~36px
+@font-size-h2: var(--font-size-h2); // ~30px
+@font-size-h3: var(--font-size-h3); // ~24px
+@font-size-h4: var(--font-size-h4); // ~18px
+@font-size-h5: var(--font-size-h5);
+@font-size-h6: var(--font-size-h5); // ~12px
```
Comment (silentsakky) [inline variables.less:230]:
```suggestion
@font-size-h6: var(--font-size-h6); // ~12px
```

**Review by miteshsavani810 — APPROVED** [review]

File: variables.less (hunk 2 above)
Comment (lavegupta) [reply variables.less]: Done

**Review by miteshsavani810 — APPROVED** [review]

**Review by k-kato — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Review by damini-vashishtha — APPROVED** [review]

**Pull request merged**